testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/RATButton.kt

```diff
@@ -14,6 +14,7 @@ import com.rakuten.tech.mobile.testapp.ui.settings.AppSettings
 class RATButton : AppCompatButton, IRatComponent {
 
     private var ratEvent: RATEvent? = null
+    private var screen_name = ""
```

Comment:
We need to set the Page Name and Site Section for all events that are sent. I think this one is never getting set to a valid value, right? For the buttons, maybe you will need to define custom XML attributes for setting these values?

Also, note that currently `RATButton` is missing the `target_ele` property - all click events are supposed to have this property. So you might need to also use a custom XML attribute for that.